hack/remote-debug/start-debugging

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+
+# Start dlv and redirect its output to /proc/1/fd/1 so that we can use
+# `oc logs -f <pod>` in a terminal (or the IDE) to read the log output
+# from openshift-router.
+dlv --listen=:7000 --api-version=2 --headless=true --accept-multiclient exec "$@" /usr/bin/openshift-router 2>&1 | tee /proc/1/fd/1
```

Comment:
`dlv` version may be older than the go version used in the binary, `--check-go-version=false` supposed to fix it. Not sure what's better - force the user to install the latest `dlv` or try to do the best effort. Note: `dlv` I installed was from RHEL repositories and it was 1.21.

Reply:
I'd rather not default to `--check-go-version=false`, but note this flag as an option to avoid `dlv` aborting if mismatched versions are detected. I'll add some commentary to the script.

Reply:
Done in d4e7766fcef7c5e7be4e0946771325f13a970e34.